src/eval.c

```diff
@@ -728,6 +730,16 @@ NULL
             addReplyError(c, "Use SCRIPT DEBUG YES/SYNC/NO");
             return;
         }
+    } else if (c->argc == 3 && !strcasecmp(c->argv[1]->ptr, "dump")) {
+        dictEntry *de;
+        luaScript *ls;
+
+        if ((de = dictFind(lctx.lua_scripts, c->argv[2]->ptr))) {
+            ls = dictGetVal(de);
+            addReplyBulk(c, ls->body);
+        } else {
+            addReplyErrorObject(c, shared.noscripterr);
```

Comment:
i guess we can also keep the old 40 fast error path

Reply:
ok